### ISSUE TITLE
Problematic Register Parsing for SPEEDSLOP

### ISSUE THREAD

**Issue by wjxway** [asukiaaa/arduino-TC78B009FTG#1]
Register 8's bits [7:0] and register 9's bits [7:2] are for SPEEDSLOP tuning, but register 8 should be the MSB instead of LSB.

**Comment by asukiaaa** [comment]
Thank you for the comment.
I updated https://github.com/asukiaaa/arduino-TC78B009FTG/commit/b194cde6bc10d25d81d8c713b3df64987ccb1079

By the way, is there any document to specify MSB?
I cannot find it on the datasheet.
https://toshiba.semicon-storage.com/info/docget.jsp?did=67874&prodName=TC78B009FTG

**Comment by wjxway** [comment]
Nope, the datasheet is not that detailed... It didn't even tell us what it the unit of P/I parameters...
